### DISCUSSION TITLE
Dedicated optimized emoji api

### DISCUSSION THREAD

**Discussion by m5o** [dicebear/dicebear#118] [Ideas]
While playing around with the `initials` api an idea popped up.
How about an api where, based on the `seed`, an unique emoji/background combination is created.

Like: `https://avatars.dicebear.com/api/initials/🤓.svg`

(Maybe bigger text by default, optimized for one single emoji character. 
E.g. `https://avatars.dicebear.com/api/initials/😂.svg?fontSize=75`)

And if the svg renders the emoji character as text, the emoji character is shown based on the user’s operating system.
(To avoid a license discussion) 

Alternative the `seed` is just a reference to the popular Emoji Cheat Sheet
https://www.webfx.com/tools/emoji-cheat-sheet/

**Inspiration**
With a nice pastel background color palette, I realize this could look similar to:

| iOS 14.4 Contacts App > Edit > Photo | iOS 14.4 Contacts App > Edit > Photo (scroll down) |
|---|---|
| ![IMG_5537](https://user-images.githubusercontent.com/26371/113992066-60487180-9853-11eb-8465-0cab3b503a93.png) | ![IMG_5538](https://user-images.githubusercontent.com/26371/113992042-5888cd00-9853-11eb-963b-1142eb93dd11.png) |

**Comment by FlorianKoerner** [comment]
There will be an avatar style with icons from Bootstrap in the upcoming version 5.0.

I think that's pretty close to your idea. With two advantages: We have no licensing problems and the icons look identical on every device.

Little preview:
https://github.com/dicebear/dicebear/tree/5.0/packages/%40dicebear/icons/tests/svg/create

<img src="https://raw.githubusercontent.com/dicebear/dicebear/5.0/packages/%40dicebear/icons/tests/svg/create/0.svg" width="120" height="120" /> <img src="https://raw.githubusercontent.com/dicebear/dicebear/5.0/packages/%40dicebear/icons/tests/svg/create/1.svg" width="120" height="120" /> <img src="https://raw.githubusercontent.com/dicebear/dicebear/5.0/packages/%40dicebear/icons/tests/svg/create/2.svg" width="120" height="120" /> <img src="https://raw.githubusercontent.com/dicebear/dicebear/5.0/packages/%40dicebear/icons/tests/svg/create/3.svg" width="120" height="120" /> <img src="https://raw.githubusercontent.com/dicebear/dicebear/5.0/packages/%40dicebear/icons/tests/svg/create/4.svg" width="120" height="120" />
